internal/integration/ogent/ent/schema/all_types.go

```diff
@@ -0,0 +1,83 @@
+package schema
+
+import (
+	"database/sql/driver"
+	"fmt"
+	"net/url"
+
+	"entgo.io/ent"
+	"entgo.io/ent/schema/field"
+	"github.com/google/uuid"
+)
+
+// AllTypes holds the schema definition for the AllTypes entity.
+type AllTypes struct {
+	ent.Schema
+}
+
+// Fields of the AllTypes.
+func (AllTypes) Fields() []ent.Field {
+	return []ent.Field{
+		field.Int("int"),
+		field.Int8("int8"),
+		field.Int16("int16"),
+		field.Int32("int32"),
+		field.Int64("int64"),
+		field.Uint("uint"),
+		field.Uint8("uint8"),
+		field.Uint16("uint16"),
+		field.Uint32("uint32"),
+		field.Uint64("uint64"),
+		field.Float32("float32"),
+		field.Float("float64"),
+		field.String("string_type"),
+		field.Bool("bool"),
+		field.UUID("uuid", uuid.Nil),
+		field.Time("time"),
+		field.Text("text"),
+		field.Enum("state").
+			Values("on", "off"),
+		// field.Strings("strings"),
+		// field.Ints("ints"),
+		// field.Floats("floats"),
+		field.Bytes("bytes"),
+		// field.JSON("nicknames", []string{}),
+		// field.JSON("json_slice", []http.Dir{}).
+		// 	Annotations(entoas.Schema(ogen.String().AsArray())),
+		// field.JSON("json_obj", url.URL{}).
+		// 	Annotations(entoas.Schema(ogen.String())),
+		// field.Other("other", &Link{}).
+		// 	SchemaType(map[string]string{dialect.Postgres: "varchar"}).
+		// 	Default(DefaultLink()).
+		// 	Annotations(entoas.Schema(ogen.String())),
+	}
```

Comment:
?

Reply:
Reminder to add support for them as well.

Reply:
I don't like reminders in the code tbh, but it's your project :)